modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -251,7 +270,20 @@ export default {
       this.editing = undefined;
     },
     updateEditing(id) {
```

Comment:
This does a lot of work. It accepts IDs to swap out media, but we also pass it `null` to clear the actively editing media.